template_example/example/standalone/providers.tf

```diff
@@ -12,7 +12,7 @@ provider "nomad" {
 
 provider "vault" {
   address = "http://127.0.0.1:8200"
-  token   = "master"
+  token   = var.vault_master_token
```

Comment:
Could you elaborate on where(how) do you set the value for this variable?

Reply:
It's basically using `TF_VAR_vault_master_token` from the box. Since Terraform imports any variable prefixed with `TF_VAR`, I figured this was a ok solution.

Reply:
I agree this works fine. But one thing I've been thinking about lately is that it is difficult to find and realise how env-variables and terraform interact, and where all of our variables come from. Maybe we could actually create a new section in the README covering it all in detail (not in this pr of course)

Reply:
I agree with @pdmthorsrud , that this setup is quite implicit from the user perspective. Might need update documentation and maybe leave a comment above variable

Reply:
I like the idea of leaving a comment above in the code, and then also creating an issue to make a section in the readme detailing variables in general (where they come from, how they are set, everything)